sunpy/net/attr.py

```diff
@@ -177,13 +212,20 @@ def update_values(cls, adict):
         >>> attrs.Instrument.aia, attrs.Instrument.hmi # doctest: +SKIP
         (<Instrument('AIA')>, <Instrument('HMI')>)
         """
-
-        for key, value in adict.items():
-            if isiterable(value) and not isinstance(value, str):
-                for pair in value:
-                    if len(pair) != 2:
-                        raise ValueError(f'Invalid length (!=2) for values: {value}.')
-                    else:
+        for client in adict.keys():
+            for key, value in adict[client].items():
```

Comment:
Can you update `key, value` to more meaningful variable names? I'm struggling to decipher what the code is doing here.

Reply:
Done.